temporal-sdk/src/main/java/io/temporal/common/converter/SearchAttributesUtil.java

```diff
@@ -0,0 +1,129 @@
+/*
+ *  Copyright (C) 2020 Temporal Technologies, Inc. All Rights Reserved.
+ *
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package io.temporal.common.converter;
+
+import io.temporal.api.common.v1.Payload;
+import io.temporal.api.common.v1.SearchAttributes;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SearchAttributesUtil {
+  private static final SearchAttributesPayloadConverter converter;
+  private static final Logger log = LoggerFactory.getLogger(SearchAttributesUtil.class);
+
+  static {
+    converter = SearchAttributesPayloadConverter.getInstance();
```

Comment:
Why do you need this? and why a separate static block is needed here?

Just use `SearchAttributesPayloadConverter.getInstance()` or `SearchAttributesPayloadConverter.INSTANCE` whereever you use it. Because everything is final - JIT will optimize this code perfectly and with ease.